src/store/plugins/observables.js

```diff
@@ -107,6 +107,18 @@ export default (store) => {
       multicast(new BehaviorSubject(new BigNumber(0))),
       refCountDelay(1000),
     ),
+    currencies: watchAsObservable(
+      ({ currencies }) => currencies,
+      { immediate: true },
+    ).pipe(
+      pluck('newValue'),
+      switchMap(() => {
+        const delay = 3600000;
+        return timer(0, delay).pipe(switchMap(() => store.dispatch('getCurrencies', delay)));
```

Comment:
I'm proposing to define a global interval (in a sore plugin or App.vue) instead, and to use `currencies` as before. https://github.com/aeternity/superhero-wallet/issues/1432